skimage/io/tests/test_simpleitk.py

```diff
@@ -8,33 +8,25 @@
 from skimage.io import imread, imsave, use_plugin, reset_plugins
 from skimage._shared import testing
 
-try:
-    import SimpleITK as sitk
-    use_plugin('simpleitk')
-except ImportError:
-    sitk_available = False
-else:
-    sitk_available = True
+from pytest import importorskip
 
-np.random.seed(0)
+importorskip('SimpleITK')
 
+np.random.seed(0)
```

Comment:
Maybe move this to `setup()`?

Reply:
you are right, this is necessary. actually, everything that uses random needs to be in a fixture...

Reply:
Moved this below, to where random numbers are generated.